contrib/windows/collie-ctl.test.ps1

```
@@ -148,6 +148,8 @@ COLLIE_HOST="127.0.0.1"
   Assert-Equal $script:registered.Principal.RunLevel "Limited" "task privilege"
   Assert-Equal $script:registered.Settings.ExecutionTimeLimit ([TimeSpan]::Zero) "task execution limit"
   Assert-Equal $script:registered.Settings.RestartCount 999 "task restart policy"
+  Assert-Contains $script:registered.Action.Execute "conhost.exe" "task launcher is headless"
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Assert-Contains' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)